modules/gdscript/gdscript.h

```diff
@@ -185,6 +185,7 @@ class GDScript : public Script {
 	virtual void update_exports();
 
 	virtual Error reload(bool p_keep_state = false);
+	void init_rpc_methods_properties();
```

Comment:
I guess this could be private and prefixed with `_`?

Reply:
Fixed.